Source/drlg_l2.cpp

```diff
@@ -5,7 +5,7 @@ int nSx2; // weak
 int nSy1;
 int nSy2; // weak
 int nRoomCnt;
-unsigned char predungeon[40][40];
+unsigned char predungeon[DMAXX][DMAXX];

```

Comment:
```diff
- unsigned char predungeon[DMAXX][DMAXX]; 
+ unsigned char predungeon[DMAXX][DMAXY]; 
```

Reply:
Very sorry about this. @AJenbo fixed it in 7c071924adbf925a496cd8827eeb045ba2272e60

Reply:
No wories @joankaradimov. Typos like this are bound to slip in. Glad we caught it quickly :)

Reply:
I'm sorry I missed it, so that's why I fixed it :) Good catch @mewmew .